extensions/wikia/WikiaMaps/WikiaMaps.i18n.php

```diff
@@ -51,8 +50,6 @@
 	'wikia-interactive-maps-create-map-choose-tile-set-tip' => 'Select an existing map template or upload your own to get started.',
 	'wikia-interactive-maps-create-map-no-tile-set-found' => 'Sorry, no templates found.',
 	'wikia-interactive-maps-create-map-clear-tile-set-search' => 'Clear search filter',
-	'wikia-interactive-maps-create-map-choose-type-tip' => 'Get started choosing a real map (ex. NYC) or custom (ex. Westeros).',
-	'wikia-interactive-maps-create-map-choose-type-tip-link' => 'Learn more',
 	'wikia-interactive-maps-create-map-title-label' => 'Name the map',
 	'wikia-interactive-maps-create-map-tile-set-title-label' => 'Name the template',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wikia-interactive-maps-create-map-tile-set-title-label" 15 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)